packages/boost/build.sh

```diff
@@ -47,6 +46,15 @@ termux_step_make_install() {
 		BOOSTAM=32
 	fi
 
+	if [ "$TERMUX_ARCH" = arm ]; then
+		#```
+		# <inline asm>:1:41: error: expected '%<type>' or "<type>"
+		# .pushsection ".debug_gdb_scripts", "MS",@progbits,1
```

Comment:
Is it possible to patch `.pushsection ".debug_gdb_scripts", "MS",@progbits,1` to `.pushsection ".debug_gdb_scripts", "MS",%progbits,1` as the comments say?

Reply:
According to the linked issue and this https://github.com/termux/termux-packages/blob/master/packages%2Fvalgrind%2Fbuild.sh#L29, yes it does fix that issue. 

Also see: https://github.com/llvm/llvm-project/issues/120871